test/gtest/cstring.cpp

```diff
@@ -143,4 +143,21 @@ TEST(cstring, literalSuffix) {
     EXPECT_TRUE((std::is_same_v<cstring, decltype(""_cs)>));
 }
 
+TEST(cstring, is_cached) {
+    cstring test = "test"_cs;
+    (void)test;
```

Comment:
I think this would be nicer:
```suggestion
    [[maybe_unused]] cstring test = "test"_cs;